boards/shields/frdm_stbc_agm01/boards/frdm_k22f.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ &arduino_i2c {
+	fxos8700@1c {
+		status = "disabled";
```

Comment:
Did you do this to make the fxos8700 sample use the shield instance? I think this might be better done as an application overlay

Reply:
>Did you do this to make the fxos8700 sample use the shield instance?

Yes, I did.

>an application overlay

This is exactly what I was missing, thanks!

Reply:
I couldn't figure out a way to apply the overlay only when both the frdm_stbc_agm01 and the frdm_k22f are selected. I removed this overlay though since the user could apply their own if they feel they need to.